client/helpers/getIcon.js

```diff
@@ -0,0 +1,30 @@
+import {
+  Achievement,
+  Anchor,
+  BusinessService,
+  Channel,
+  Dashboard,
+  Deploy,
+  FingerPrint,
+  Group,
+  Grow,
+  Organization
+} from 'grommet-icons'
+
+export default function getIcon () {
+  const array = [
+    Achievement,
+    Anchor,
+    BusinessService,
+    Channel,
+    Dashboard,
+    Deploy,
+    FingerPrint,
+    Group,
+    Grow,
+    Organization
+  ]
+
+  const iconName = array[Math.floor(Math.random() * 9)]
```

Comment:
```suggestion
  const iconName = array[Math.floor(Math.random() * (array.length - 1))]
```

💭 If we later decide to add another icon to `array`, we won't have to remember to manually increment the magic number in this expression.